core/src/main/resources/lib/form/hetero-list.jelly

```diff
@@ -82,31 +82,27 @@ THE SOFTWARE.
     <d:tag name="body">
       <div style="width:100%">
         <j:set var="help" value="${descriptor.helpFile}" />
-        <j:if test="${hasHeader}">
-          <div class="tr help-sibling">
-            <div colspan="3">
-              <div class="${attrs.disableDragAndDrop or readOnlyMode ? '' : 'dd-handle'}">
-                <b>${descriptor.displayName}</b>
-              </div>
-              <f:helpLink url="${help}"/>
-            </div>
-          </div>
-          <!-- TODO: help support is unintuitive; people should be able to see help from drop-down menu -->
-          <j:if test="${help!=null}">
-            <f:helpArea />
+
+        <div class="repeated-chunk__header">
```

Comment:
might be this or something else but when creating a new agent there's empty space showing:

![image](https://user-images.githubusercontent.com/21194782/142167611-73c1f1bb-8748-4686-99bc-fdaf93aa8867.png)